src/VisualStudio/Core/Def/Implementation/TaskList/ProjectExternalErrorReporter.cs

```diff
@@ -27,6 +27,12 @@ internal class ProjectExternalErrorReporter : IVsReportExternalErrors, IVsLangua
 
         private readonly VisualStudioWorkspace _workspace;
         private readonly ExternalErrorDiagnosticUpdateSource _diagnosticProvider;
+        
+        // Constructor used by F# kept for backwards compatability
```

Comment:
There's a pattern you'll see in other places where I'll write something like `[Obsolete("This is a compatibility shim for F#; please do not use it.")]` This prevents others from accidentally using this one again, and means once F# moves to newer stuff they'll also know to move off of this.

(100% OK with this being fixed later since this is a blocking and urgent issue.)